src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -144,30 +176,16 @@ internal void AddProvider(string workerDir)
 
                     _logger.LogDebug("Found worker config: {workerConfigPath}", workerConfigPath);
 
-                    var workerConfig = GetWorkerConfigJsonElement(workerConfigPath);
-                    var workerDescriptionElement = workerConfig.GetProperty(WorkerConstants.WorkerDescription);
```

Comment:
All these lines (148-150 and 152-170) are moved to a method `WorkerConfigurationHelper.GetWorkerDescription()`